jupyterlab_git/git.py

```diff
@@ -555,14 +570,26 @@ async def detailed_log(self, selected_hash, path):
         Execute git log -1 --numstat --oneline -z command (used to get
         insertions & deletions per file) & return the result.
         """
-        cmd = ["git", "log", "-1", "--numstat", "--oneline", "-z", selected_hash]
-        code, my_output, my_error = await execute(
-            cmd,
-            cwd=path,
-        )
+        cmd = [
+            "git",
+            "log",
+            "-1",
+            "--numstat",
+            "--oneline",
+            "-z",
+            selected_hash,
+        ]
 
-        if code != 0:
-            return {"code": code, "command": " ".join(cmd), "message": my_error}
+        commit_cmd = [
+            "git",
+            "log",
+            "-1",
+            "--pretty=format:%B",
+            selected_hash,
+        ]
+
+        code, my_output = await execute_log_command(cmd, path)
+        my_commit_output = await execute_log_command(commit_cmd, path)
```

Comment:
Playing a bit with the command, using the following single command:

`git log -1 --numstat --pretty=format:%B%x00 -z <hash>` will give you all required information at once. The first item will be the subject + body.

> Shouldn't you use only `%b` to get the body and not the subject?

Reply:
I ended up using `git log -1 --numstat --pretty=format:%b%x00 -z <hash>`.